ts/types/wallet.ts

```diff
@@ -1,91 +1,63 @@
-import { ImageSource } from "react-native-vector-icons/Icon";
+import { fromEither, fromNullable } from "fp-ts/lib/Option";
+import {
+  AmountInEuroCents,
+  AmountInEuroCentsFromNumber
+} from "italia-ts-commons/lib/pagopa";
+import { Amount } from "../../definitions/pagopa/Amount";
+import { Psp } from "../../definitions/pagopa/Psp";
+import { Transaction } from "../../definitions/pagopa/Transaction";
+import {
+  UNKNOWN_AMOUNT,
+  UNKNOWN_DATE,
+  UNKNOWN_NUMBER,
+  UNKNOWN_PAYMENT_REASON,
+  UNKNOWN_STRING
+} from "./unknown";
+
+const extractAmount = (amountObject: Amount | undefined): AmountInEuroCents =>
```

Comment:
this [looks like a lens](https://github.com/gcanti/monocle-ts#frompath)

Reply:
actually [this](https://github.com/gcanti/monocle-ts#fromnullableprop) may be what you need